pwnlib/util/fiddling.py

```diff
@@ -615,14 +616,15 @@ def hexdump_iter(fd, width=16, skip=True, hexii=False, begin=0, style=None,
         total = fd.len
     else:
         # Save the current file offset
-        cur = fd.seek(0, os.SEEK_CUR)
+        cur = fd.tell()
 
         # Determine the total size of the file
         fd.seek(0, os.SEEK_END)
-        total = fd.tell()
+        total = fd.tell() - cur
+
+        # Restore the file offset, and
```

Comment:
No "and", just a dangling lost thought in a comment.

This commit is cherry-picked from `dev` (f46112e8), which has the same dangling comment -- but also has tests which pass.